### DISCUSSION TITLE
Doing pagination based on infinite scrolling

### DISCUSSION THREAD

**Discussion by noghartt** [KevinVandy/material-react-table#33] [Q&A] [answered]
How we could implement pagination based on infinite scrolling with `material-react-table`.

As I saw, the lib abstract a lot of things from `useVirtualizer` and `useReactTable`, how we could handle these abstraction and get access to things like: `rowVirtualizer.getVirtualItems()` and `table.getRowModel()`, that is essential to handle infinite scrolling fetch?

**Comment by KevinVandy** [comment, marked as answer]
Documenting the official MRT example lives here https://www.material-react-table.com/docs/examples/infinite-scrolling
